src/avg.js

```diff
@@ -10,14 +10,23 @@ let sum = require('./sum');
  * @returns {number} The average
  */
 function avg(array) {
+  let count=0;
+  for (let i in array) {
```

Comment:
Use `for (let i = 0; i < array.length; i++) { ... }` to iterate over the indexes of an array.

Use `for (let value of array) { ... }` to iterate over just the values (if you don't need the indexes).

Don't ever use `for (let ___ in ____)`. Pretend it doesn't exist.